hnn_core/viz.py

```diff
@@ -419,22 +419,22 @@ def plot_tfr_morlet(dpl, *, freqs, n_cycles=7., tmin=None, tmax=None,
     return ax.get_figure()
 
 
-def plot_spectrogram(dpl, *, fmin, fmax, winlen=None, tmin=None, tmax=None,
-                     layer='agg', ax=None, show=True):
-    """Plot Welch spectrogram (power spectrum) of dipole time course
+def plot_periodogram(dpl, *, fmin, fmax, tmin=None, tmax=None, layer='agg',
```

Comment:
What about `plot_psd()` to be explicit that we're plotting spectral density in units of power rather than e.g. spectral mass in units of energy?